test/drivers/event_class/event_class.cpp

```diff
@@ -155,6 +155,11 @@ event_test::event_test(int syscall_id, int event_direction):
 			m_event_type = PPME_GENERIC_X;
 		}
 
+		/* This logic is not needed by architectures that require the `CAPTURE_SCHED_PROC_FORK`
+		 * workaround, since `CAPTURE_SCHED_PROC_FORK` requires `BPF_RAW_TRACEPOINTS` and so
+		 * kernel versions >= 4.17
+		 */		
+#ifndef CAPTURE_SCHED_PROC_FORK  
```

Comment:
Shouldn't we check the kernel version or if has raw tracepoints here?

Reply:
Uhm right now we have this check at compile time in the probe bpf https://github.com/falcosecurity/libs/blob/master/driver/bpf/quirks.h#L31-L33... maybe we can do something better a build time like if you enable `-DBUILD_BPF` you should have a kernel version `>=4.17` on some arch like `aarch64` and `s390x`. BTW this doesn't seem too much related to this test PR, we can do that in another PR when we decide how to proceed, WDYT?

Reply:
I think this is more a follow-up as we should then review and consider the other options.